### PR TITLE
SkullBlock.java

### PR DESCRIPTION
**Pull request by chaunhan**
Skull :3

### CODE REVIEW

**Comment by Creeperface01** [comment]
you have not added SKULL variable and tile

**Comment by Creeperface01** [comment]
so you should create one pull instead of 6

File: src/main/java/cn/nukkit/block/SkullBlock.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import cn.nukkit.nbt.tag.StringTag;
+import cn.nukkit.Player;
+import cn.nukkit.tile.Tile;
+import cn.nukkit.math.AxisAlignedBB;
+import cn.nukkit.nbt.tag.ByteTag;
+import cn.nukkit.tile.Skull;
+
+
+public class SkullBlock extends Transparent{
+	public static final int SKELATON_SKULL = 0;
```
Comment (xpyctum) [inline src/main/java/cn/nukkit/block/SkullBlock.java]: tab

**Comment by Snake1999** [comment]
Do one pull request instead of many next time. But thanks for your works, these will add to Nukkit soon :D

**Pull request closed without merging**